rust/ql/src/change-notes/2025-11-07-example-queries.md

```diff
@@ -0,0 +1,4 @@
+---
+category: newQuery
```

Comment:
I would've thought `newQuery` meant a new query meant for running by end users. Though I don't know what other category we could pick, so maybe this is the best option?

Reply:
Thanks for the ping. What are the available categories?

Reply:
Thanks @geoffw0 -` newQuery` looks great to me 👍🏻

Reply:
Great, thanks for taking a look.